<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmpresaTeis</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/gabrielgsd.jpg</normaloff>../img/gabrielgsd.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="pan_main">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_customer">
       <attribute name="title">
        <string>Customers</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0" columnstretch="0,0,0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0,0,0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="2" column="3">
             <widget class="QLabel" name="lbl_province">
              <property name="text">
               <string>Province:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QLineEdit" name="le_address">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>9999</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_address">
              <property name="text">
               <string>Address:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="7">
             <widget class="QCheckBox" name="chkb_hystorical">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Hystorical</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4" colspan="2">
             <widget class="QLineEdit" name="le_email">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>99999</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="7">
             <widget class="QComboBox" name="cb_city">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lbl_invoice">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Invoice Type:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="7">
             <widget class="QLineEdit" name="le_name">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="4" colspan="2">
             <widget class="QLineEdit" name="le_surname">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>99999</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="lbl_surname">
              <property name="text">
               <string>Surname:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QLabel" name="lbl_name">
              <property name="text">
               <string>Name:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QRadioButton" name="rb_paper">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>paper billing</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">grp_invoice</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="2" alignment="Qt::AlignVCenter">
             <widget class="QPushButton" name="btn_search">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string> Search</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../img/glass-icon.svg</normaloff>../img/glass-icon.svg</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QRadioButton" name="rb_electronic">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>e-invoice</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">grp_invoice</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="6">
             <widget class="QLabel" name="lbl_city">
              <property name="text">
               <string>City:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_date">
              <property name="text">
               <string>Register Date:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_dni">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_date">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="6">
             <widget class="QLabel" name="lbl_status">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="6">
             <widget class="QLabel" name="lbl_name_2">
              <property name="text">
               <string>Phone:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="btn_calendar">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../img/calendar.png</normaloff>../img/calendar.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="7">
             <widget class="QLineEdit" name="le_phone">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="lb_email">
              <property name="text">
               <string>Email:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbl_dni">
              <property name="text">
               <string>DNI/CIF:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4" colspan="2">
             <widget class="QComboBox" name="cb_province">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>999999</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btn_save_cust">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>99999</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_modify_cust">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>99999</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Modify</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_del_cust">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>99999</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>200</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_clear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lbl_list_cust">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1999999</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Customer List</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="table_customer">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>999999</width>
              <height>401</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Surname</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Mobile</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Province</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>City</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Invoice Type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Status</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_Data">
     <property name="title">
      <string>Export Data (.xsl)</string>
     </property>
     <addaction name="actionExportCustomers"/>
    </widget>
    <addaction name="actionSettings"/>
    <addaction name="menuExport_Data"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBackup"/>
    <addaction name="actionRestoreBackup"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
    <addaction name="actionCustomerReport"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="dlg_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuReports"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBackup"/>
   <addaction name="actionRestoreBackup"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/exit.svg</normaloff>../img/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCustomerReport">
   <property name="text">
    <string>Customer Report</string>
   </property>
  </action>
  <action name="dlg_about">
   <property name="text">
    <string>About </string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="icon">
    <iconset>
     <normaloff>../img/backup.svg</normaloff>../img/backup.svg</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionRestoreBackup">
   <property name="icon">
    <iconset>
     <normaloff>../img/backup-restore.svg</normaloff>../img/backup-restore.svg</iconset>
   </property>
   <property name="text">
    <string>Restore Backup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>../img/setting.svg</normaloff>../img/setting.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExportCustomers">
   <property name="text">
    <string>Customers</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>le_date</tabstop>
  <tabstop>btn_calendar</tabstop>
  <tabstop>le_surname</tabstop>
  <tabstop>le_name</tabstop>
  <tabstop>le_dni</tabstop>
  <tabstop>le_email</tabstop>
  <tabstop>le_phone</tabstop>
  <tabstop>le_address</tabstop>
  <tabstop>cb_city</tabstop>
  <tabstop>rb_electronic</tabstop>
  <tabstop>rb_paper</tabstop>
  <tabstop>chkb_hystorical</tabstop>
  <tabstop>btn_save_cust</tabstop>
  <tabstop>btn_modify_cust</tabstop>
  <tabstop>btn_del_cust</tabstop>
  <tabstop>table_customer</tabstop>
  <tabstop>pan_main</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="grp_invoice"/>
 </buttongroups>
</ui>
